gymnasium/envs/tabular/blackjack.py

```diff
@@ -0,0 +1,477 @@
+"""This module provides a Blackjack functional environment and Gymnasium environment wrapper BlackJackJaxEnv."""
+
+
+import math
+import os
+from typing import Optional, Tuple, Union
+
+import jax
+import jax.numpy as jnp
+import numpy as np
+from jax import random
+from jax.random import PRNGKey
+
+from gymnasium import spaces
+from gymnasium.error import DependencyNotInstalled
+from gymnasium.experimental.functional import ActType, FuncEnv, StateType
+from gymnasium.experimental.functional_jax_env import FunctionalJaxEnv
+from gymnasium.utils import EzPickle, seeding
+from gymnasium.wrappers import HumanRendering
+
+
+RenderStateType = Tuple["pygame.Surface", str, int]  # type: ignore  # noqa: F821
+
+
+deck = jnp.array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 10, 10, 10])
+
+
+def obs_from_device(obs):
```

Comment:
If we are using a box obs spaec, is there any need for this change?

Reply:
removed